testing/log4cplus/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: Sven-Hendrik Haase <svenstaro@gmail.com>
+# Contributor: Alexey Galakhov <agalakhov at gmail dot com>
+# Contributor: Andreas Baumann <abaumann at yahoo dot com>
+# Contributor: Steve Holweg <skytep@gmail.com>
+
+pkgname=log4cplus
+pkgver=1.2.0
+pkgrel=1
+pkgdesc="A C++ logger very close to Java's log4j"
+arch="all"
+url="http://log4cplus.sourceforge.net/"
+license="GPL"
```

Comment:
Please use SPDX name of the license. GPL is too vague, there are many versions and variants of GPL.